docs/collections/documents/geospatial/tutorial.md

```diff
@@ -75,130 +75,137 @@ Let's assume your
-    radius = 10 * 1000 # meters
-    print("\n3a. FIND_NEAR_BY_SHOPS for request: ({}, {})".format(lat1, long1))
-    within_query = "FOR loc IN WITHIN ({}, {}, {}, {}) SORT loc._distance ASC RETURN loc".format(collection_shops, lat1, long1, radius)
-    cursor = client.execute_query(within_query)
-    docs = [document for document in cursor]
-    pp.pprint(docs)
-
-  # Step3b: Find shop closest to given location
-    print("\n3b. FIND_NEAREST_SHOP for request: ({}, {})".format(lat1, long1))
-    near_query = "FOR loc IN NEAR ({}, {}, {}, 1) RETURN loc".format(collection_shops, lat1, long1)
-    cursor = client.execute_query(near_query)
-    docs = [document for document in cursor]
-    pp.pprint(docs)
-
-  #-------------------------------------------------------------------#
-
-    # Step2: Create a collection & geoindex if not exists
-    print("\n4. CREATE_COLLECTION: region: {},  collection: {}".format(global_url, collection_transit))
-    if client.has_collection(collection_transit):
-        transitCol = client.collection(collection_transit)
-    else:
-        transitCol = client.create_collection(collection_transit)
-        client.add_geo_index(collection_transit, fields=["coordinate"], ordered=False)
-
-    # Step5: Insert data into collection.
-    print("\n5. INSERT_TRANSIT_DATA: in region: {}".format(global_url))
-    client.insert_document(collection_name=collection_transit, document=transit_data)
-
-  # Step6a: Find buses currently within given rectangular fence
-    lat1 = 37.38905593900322 
-    long1 = -122.14426630984782 
-    lat2 = 37.332401582858324 
-    long2 = -121.80235913612003 
-    
-    print("\n6a. FIND_BUSES available within RECTANGULAR fence: ({}, {}, {}, {}) sorted by distance".format(lat1, long1, lat2, long2))
-    within_query = "FOR loc IN WITHIN_RECTANGLE ({}, {}, {}, {}, {}) SORT loc._distance ASC RETURN loc".format(collection_transit, lat1, long1, lat2, long2)
-    cursor = client.execute_query(within_query)
-    docs = [document for document in cursor]
-    pp.pprint(docs)
-
-  # Step6b: Find shop closest to given location
-    print("\n6b. FIND_NEAREST_BUS for given location: ({}, {})".format(lat1, long1))
-    near_query = "FOR loc IN NEAR ({}, {}, {}, 1) RETURN loc".format(collection_transit, lat1, long1)
-    cursor = client.execute_query(near_query)
-    docs = [document for document in cursor]
-    pp.pprint(docs)
-
-  #-------------------------------------------------------------------#
-  
-    # # Step7: Delete Data
-    print("\n7. DELETE_DATA: region: {}, collections: {}, {}".format(global_url, collection_shops, collection_transit))
-    shopCol.truncate()
-    transitCol.truncate()
-    #client.delete_collection(collection_name)
+""" This file is a demo of using Geo Locations on collections """
+import pprint
+from c8 import C8Client
+
+# Variables - Connection
+URL = "gdn.paas.macrometa.io"
+API_KEY = "my API key" #Change to my API key
```

Comment:
we should specify the apikey sample as API_KEY = "apikey <yourapikey>" since "apikey" keyword is needed here and the user doesn't know that.

Reply:
We should probably test this together.
For me, it worked fine without the "apikey" keyword.